.github/workflows/kvrocks.yaml

```diff
@@ -88,11 +88,11 @@ jobs:
 
       - name: Install Dependencies
         run: |
-          brew install gcc autoconf automake libtool cmake
+          brew install autoconf automake libtool cmake
           mkdir build
 
       - name: Build
         run: |
           cd build
-          cmake ..
+          cmake .. -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++
```

Comment:
So can we remove CMAKE_CXX_COMPILER/CMAKE_C_COMPILER? since we can workaround it when failing to build.

Reply:
done

Reply:
seems some problem here, I will check it